README.md

```diff
@@ -61,14 +73,17 @@ This will pull everything needed and build release. Then you can play with code
 Building Docker image
 ---------------------
 
+If for any reasons you want better control over Docker containers please use the
+following commands after editing files as you wish:
```

Comment:
@krystian-hebel do not write like that. User should not modify Dockerfiles, as it can impact the build process and even lead to platform damage in worst cases. It is not about better control, it is about the awareness what has been used to build containers.

Reply:
Then I don't see a reason for having this paragraph in README in the first place. Why show user how to build a container if he/she is not supposed to?

Reply:
@krystian-hebel user not supposed to edit Dockerfiles. I did not say user is not allowed to build container. Modifying Dockerfile and building container is a separate thing.

Reply:
@miczyg1 I stand corrected.